index.js

```diff
@@ -310,19 +324,76 @@ export function buildJsx(tree, options = {}) {
       }
 
       if (automatic) {
-        if (children.length > 1) {
+        parameters.push(props || {type: 'ObjectExpression', properties: []})
+
+        if (key) {
+          parameters.push(key)
+        } else if (options.development) {
+          parameters.push({
+            type: 'UnaryExpression',
+            operator: 'void',
+            prefix: true,
+            argument: {type: 'Literal', value: 0}
+          })
```

Comment:
I don’t have strong opinions on this. Both TypeScript and Babel emit `void 0`, so that’s closer to existing tools. Just let me know if you want me to change it.

Reply:
It’s not very important, but I think it’s only needed for a really esoteric case, that `undefined` is not `undefined`: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/undefined, or for mangling. Given that we try to generate reasonable code with names, I do prefer changing it to `undefined`.